sdk/eventhub/Azure.Messaging.EventHubs/Directory.Build.props

```diff
@@ -12,6 +12,10 @@
     <IsTestSupportProject>true</IsTestSupportProject>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
@chidozieononiwu:  Can you help me understand the purpose of this and why we're applying it to some services and not others?